src/Timer.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+
+class Timer extends React.Component{
+    constructor(props){
+        super(props);
+        this.state = {date: new Date()};
+        this.futureDate = {future: new Date('December 20, 2019 ')};
+        this.elapsed = this.futureDate.future.getTime() - this.state.date.getTime()
```

Comment:
Maybe think about putting these in your state? `this.state = { date, futureDate, elapsed }` Would change some of your other code though. Instead of `this.futureDate.future` it would be `this.state.futureDate`